lib/hooks/useTime.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { formatter } from '../utils';
+
+const useTime = () => {
+  const SunCalc = require('suncalc');
+  const [currentTime, setCurrentTime] = useState(formatter.format(Date.now()));
+  const [currentTimeNoFilter, setCurrentTimeNoFilter] = useState(Date.now());
+  const [isDaytime, setIsDaytime] = useState(false);
```

Comment:
Since this stats variable is set upon every update to `currentTimeNoFilter` and is strictly dependent on that value, and that value itself will update every render due to the useEffect, perhaps you don't need this state variable.

Instead, you can opt to compute `isDaytime` right here in the body of the hook, and simply return it in the list at the bottom.